chapters/VK_KHR_surface/wsi.txt

```diff
@@ -1061,6 +1061,13 @@ This has been updated to indicate that it uses RGB color encoding, not XYZ.
 The old name is deprecated but is maintained for backwards compatibility.
 ====
 
+[NOTE]
+.Note
+====
+For a linear or non-gamma transfer function color space use
+VK_COLOR_SPACE_PASS_THROUGH_EXT
```

Comment:
I missed the discussion on this, so apologies if this has already been discussed, but I think it would be better to say, for example
> For a linear or non-gamma transfer function color space with unknown color primaries use ...

because there are other linear transfer function color spaces that would not use PASS_THROUGH like ADOBERGB_LINEAR.

Reply:
We had an initial discussion with some of our experts in formats. But we haven't ended our discussion. This was the first proposal, with expectation of further feedback.
You have a good point, I will bring this us at next discussion. Thank you